<!-- Spawns a tracker for a given object -->
<launch>


<!-- Run all the normal processing stuff but tell openni.launch not to
actually get any data from a real kinect (that will be supplied by the bag -->

	<group ns="/tracking">
		<node pkg="race_3d_object_tracking" type="object_tracking_node" name="$(arg object_id)" output="screen" required="true">
		<param name="parent_frame_id" value="$(arg parent_frame_id)"/>
		<param name="pc_in" value="$(arg pc_in)"/>
		<param name="color_r" value="$(arg color_r)"/>
		<param name="color_g" value="$(arg color_g)"/>
		<param name="color_b" value="$(arg color_b)"/>
		<param name="color_a" value="$(arg color_a)"/>
		<param name="delta" value="$(arg delta)"/>
		<param name="maxparticlenum" value="$(arg maxparticlenum)"/>
		<param name="epsilon" value="$(arg epsilon)"/>
		<param name="bin" value="$(arg bin)"/>
		<param name="object_pcd_file" value="$(arg object_pcd_file)"/>
		<!--<rosparam file="$(find race_3d_object_tracking)/config/tracker_default_params.yaml" command="load" />-->
		</node>
	</group>
</launch>	
